C++/day15.cpp

```diff
@@ -0,0 +1,56 @@
+#include <iostream>
+#include <cstddef>
+using namespace std;	
+class Node
+{
+    public:
+        int data;
+        Node *next;
+        Node(int d){
+            data=d;
+            next=NULL;
+        }
+};
+class Solution{
+    public:
+    Node* insert(Node *head,int _data) {
+        Node* _new = new Node(_data);
+        _new->next = NULL;
+        if (head == NULL) {
+            // don't forget the initial case!
+            // head is NULL b.a.e. on first run
+            head = _new;
+        } else if (head->next == NULL) {
```

Comment:
Isn't this case already handled by the following `else`? If `head->next == NULL` then the `while` loop will never be executed and `start` will be equal to `head`